### DISCUSSION TITLE
Replace notes entry with daily habit diary entry

### DISCUSSION THREAD

**Discussion by javascriptisbest** [iSoron/uhabits#1242] [Feature Requests]
I think the question for you entry is enough for us to wirte a note or a motivational word. So it would be better if we replace it with a nice little diary, like in the Ticktick app. It helps us to record sad and happy memories and milestones achieved with our habits. That must be a great source of movation to keep going. Thank you for taking the time to read my request. I am not good English so please forgive me if i say anything wrong

**Comment by herrbrixter** [comment]
Duplicate of #2253
